src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -131,50 +132,54 @@ export default function AttachmentsOverview() {
   }
 
   return (
-    <div
-      className={styles.overview}
-      onPaste={e => handleUpload(e, e.clipboardData.files)}
-      onDrop={e => handleUpload(e, e.dataTransfer.files)}
-      onDragOver={handleDragOver}
-    >
-      <div className={styles.upload}>
-        <form className={styles.uploadForm}>
-          <label htmlFor="fileUpload" className={styles.uploadLabel}>
-            <Trans i18nKey="attachmentUpload">
-              Attach files by dragging &amp; dropping, selecting or pasting
-              them.
-            </Trans>
-          </label>
-          <input
-            type="file"
-            id="fileUpload"
-            multiple
-            onChange={e => handleUpload(e, e.target.files)}
-          />
-        </form>
-        <CameraUpload onNewAttachment={handleNewAttachment} />
-      </div>
-      {getSelectedImages().length !== 0 && (
-        <div className={styles.actions}>
-          <button
-            onClick={deleteSelected}
-            className={`omrs-btn omrs-filled-action`}
-          >
-            <Trans i18nKey="deleteSelected">Delete selected</Trans>
-          </button>
+    <UserHasAccessReact privilege="View Attachments">
```

Comment:
@jonathandick so @ivange94 is suggesting that we introduce a pair view + edit privileges tentatively named:
* `"View Attachments"`
* `"Create Attachments"`

But we need to come up with some sort of convention here right?

Reply:
I like the pair view + edit because this sticks to the OpenMRS data model, are we ok to try to stick to this across MF?